Dockerfile.devel

```diff
@@ -12,7 +12,7 @@ ENV APP_ROOT=/opt/app-root \
 
 RUN sed -i s/enabled=./enabled=0/g /etc/yum/pluginconf.d/subscription-manager.conf
 
-RUN yum upgrade -y 
+RUN yum upgrade -y
```

Comment:
When there are changes in the devel image, a new image should be pushed to the registry and the [circle-ci config](https://github.com/3scale/APIcast/blob/a4f11234e56035d3c8d152a6f26e77a8093e6cee/.circleci/config.yml#L115) and [docker compose files](https://github.com/3scale/APIcast/blob/a4f11234e56035d3c8d152a6f26e77a8093e6cee/docker-compose-devel.yml#L5) should be updated.

Currently the image url+tag is `quay.io/3scale/apicast-ci:openresty-1.19.3`. `1.19.3` makes reference to the openresty version. I suggest a new tag

```
quay.io/3scale/apicast-ci:openresty-1.19.3-01
```